internal/services/postgres/postgresql_flexible_server_resource.go

```diff
@@ -384,11 +384,6 @@ func resourcePostgresqlFlexibleServer() *pluginsdk.Resource {
 			newMb = newStorageMbRaw.(int)
 			newTier = newTierRaw.(string)
 
-			// storage_mb can only be scaled up...
```

Comment:
is it not possible to also detect that something has changed forcing a new resoruce to be created and just not throw the error?

Reply:
Unfortuanately, I didn't find any way to ask terraform if the resource is going to be destroyed or not, I assume that bit is only known by the core runtime. So I have to move it to create/update func.

Reply:
👍